src/components/richTextInput.js

```diff
@@ -124,6 +122,18 @@
     const helperTextResolved = useText(helperText);
     const { current: labelControlRef } = useRef(generateUUID());
 
+    const sanitizeHTML = (rawString) => {
+      // Replace all the whitespace between tags with nothing. Replacement is done when there is ONLY whitespace
+      return rawString
+        .toString()
+        .trimLeft()
+        .trimRight()
```

Comment:
Can't this just be `trim()`?

Reply:
Totally forgot to do this change; Suggestion has been applied